charts/pravega/values.yaml

```diff
@@ -24,3 +24,5 @@ pravega:
   storage:
     className: standard
   tier2: pravega-tier2
+
+testmode: true
```

Comment:
We don't need to add a new variable to the pravega chart here. The pravega-operator has a "testmode" variable, if that is set to true, we should use the test config map yaml else use the prod one.
https://github.com/pravega/pravega-operator/blob/master/charts/pravega-operator/values.yaml#L24